cli/src/semgrep/commands/ci.py

```diff
@@ -155,6 +155,7 @@ def fix_head_if_github_action(metadata: GitMeta) -> None:
     is_flag=True,
     hidden=True,
 )
+@click.option("--code", is_flag=True, hidden=True)
```

Comment:
should we start to think of a more general approach like --products=code,sca instead of a flag for each product?

Reply:
I definitely think we should, doesn't seem to be what @milanwilliams is thinking is needed right now. You two should talk.

Reply:
there has been a lot of discussion on this but people seem divided, so I think better to leave as-is in the absence of agreement to bother changing it